src/components/pages/use-case/content/content.jsx

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable react/no-unstable-nested-components */
 /* eslint-disable no-underscore-dangle */
 import { PortableText } from '@portabletext/react';
+import Code from 'components/shared/code';
```

Comment:
eslint reordered it this way.

Reply:
We pulled these changes into another branch and PR. Closing this one.